### DISCUSSION TITLE
Groups

### DISCUSSION THREAD

**Discussion by iambenmitchell** [upptime/upptime#474] [Ideas]
Pretty self explanatory I think.

Basically the ability to group different services together. For example you might have

```
Android App/ 🟩
├─ Android Database 🟩
├─ Google play services 🟩
├─ Sign in with Facebook 🟩

Mail Servers/ 🟩
├─ MX Germany 🟩
├─ MX USA 🟩
├─ MX AUS 🟩

iOS App/ 🟩
├─ Apple Notifications 🟩
├─ iOS Database 🟩
├─ Sign in with Apple 🟩

Website/ 🟧
├─ DNS Server 🟥
├─ Web Server 🟩

```

This would make the status website very tidy and neat. You'll be able to group services together such as website services or app services.

**Comment by se-bastiaan** [comment]
This is basically #121
